addons/pos_self_order/__manifest__.py

```diff
@@ -69,6 +69,7 @@
             "point_of_sale/static/src/app/generic_components/**/*",
             "point_of_sale/static/src/css/pos_receipts.css",
             "point_of_sale/static/src/app/screens/receipt_screen/receipt/**/*",
+            "pos_self_order/static/src/overrides/components/receipt_header/*",
```

Comment:
why don't we import the whole `/src` folder?

Reply:
Because some overrides target the point_of_sale assets and not the pos_self_order assets

Reply:
you are right! My mistake!